src/documentation/markdown/2019/filing-faq.md

```diff
@@ -68,69 +67,64 @@ Additionally, if you are using Internet Explorer, we have found that filers have
 - ffiec.cfpb.gov
 - ffiec.beta.cfpb.gov
 
-
-#### If I use separate systems to create my LAR, will uploading more than one at a time be an issue?
+### If I use separate systems to create my LAR, will uploading more than one at a time be an issue?
 If using two separate systems to create your LAR File, please combine all entries into one file. Uploading a new file will overwrite the first file uploaded.
 
+## Submission
 
-### Submission
-
-#### March 1, 2020 falls on a Sunday, what is the filing deadline for submission of 2019 HMDA data?
+### March 1, 2020 falls on a Sunday, what is the filing deadline for submission of 2019 HMDA data?
 <a target="_blank" rel="noopener noreferrer" href="https://www.consumerfinance.gov/policy-compliance/rulemaking/regulations/1003/5/#a-1">As stated in Section 1003.5(a)(1)(iii)</a> when the last day for submission falls on a Saturday or Sunday a submission shall be considered timely if it is submitted on the next succeeding Monday. Submissions of 2019 HMDA data will be considered timely if received on or before Monday, March 2, 2020.
 
-#### When is the filing period?
+### When is the filing period?
 The **Annual filing** period is between <a target="_blank" rel="noopener noreferrer" href="https://ffiec.cfpb.gov/documentation/2019/annual-filing-dates/">January 1 and March 1</a> of each year.  The <a target="_blank" rel="noopener noreferrer" href="https://ffiec.cfpb.gov/filing">HMDA Filing Platform</a> for each filing period opens January 1st.  The submission deadline is March 1st.  
 
-#### How do I know my file was submitted?
+### How do I know my file was submitted?
 Upon signing and submitting your file, an email confirming your submission will be sent to the email address on file. Additionally, when you log into your account, you should see a message stating:
 "Your submission has been accepted. This completes your HMDA filing process for this year. If you need to upload a new HMDA file, the previously completed filing will not be overridden until all edits have been cleared and verified, and the new file has been submitted." 
 Below that message is a button that says 'View Completed Filing'. Select that and you will see the date and timestamp your file was accepted along with a receipt number for the submission. 
 
-#### When can I submit my file?
+### When can I submit my file?
 The <a target="_blank" rel="noopener noreferrer" href="https://ffiec.beta.cfpb.gov/) (https://ffiec.beta.cfpb.gov/">HMDA Beta Filing Platform</a> is open throughout the year to test your HMDA data.
 
-_Annual HMDA data submissions_ for the 2019 filing year are accepted January 1 - March 1, 2020.  
+_Annual HMDA data submissions_ for the filing year are accepted between January 1 – March to be considered timely. 
```

Comment:
1) I think we still want the 1 in `March 1`
2) The current text sounds awkward to me.  What do you of this alternative?:
  _Annual HMDA data submissions_ for the filing year are considered timely when completed between January 1st and March 1st. 
```suggestion
_Annual HMDA data submissions_ for the filing year are accepted between January 1 – March 1 to be considered timely. 
```

Reply:
I agree with you on changing it. I have updated the markdown files to say `Annual HMDA data submissions for the filing year are considered timely when completed between January 1st and March 1st`.